tests/functional_tests/test_groups/models/qwen_vl/test_qwen3_vl_conversion.py

```
@@ -464,7 +472,7 @@ def test_moe_conversion(self, qwen3_vl_moe_toy_model_path, tmp_path, tp, pp):
         if result.returncode != 0:
             print(f"STDOUT: {result.stdout}")
             print(f"STDERR: {result.stderr}")
-            assert False, f"MoE conversion failed with return code {result.returncode}"
+            assert False, f"MoE {test_name} conversion failed with return code {result.returncode}"
```

Ruff (0.15.9)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)